runtime/tools/python/ttrt/common/perf.py

```diff
@@ -17,11 +17,16 @@
 import atexit
 import traceback
 from pathlib import Path
+import csv
 
 from ttrt.common.util import *
 from ttrt.common.query import Query
 
 
+def get_loc_data_hook(binary, programContext, opContext):
+    op_debug_str = ttrt.runtime.get_op_debug_str(opContext)
```

Comment:
I'm assuming we aren't using the hook to get the loc info, but just querying the Tracy message itself? Since this function is not called anywhere.

Reply:
This keeps getting rebased in oh my god that's super annoying. I'll add a branch to drop it.